[ADX-286] Origin/ai merge

AbomDocPhibes opened the pull request:
version 4.0.2 included AI and "simple" prompt functionality and scipt settings to Open ADC

AbomDocPhibes committed:
added prompt script

AbomDocPhibes committed:
Update changelog.md

AbomDocPhibes committed:
Update config.xml

AbomDocPhibes committed:
Update config.xml

AbomDocPhibes committed:
Update Open.adc

AbomDocPhibes committed:
Update open.html

AbomDocPhibes committed:
Update Open.adc

AbomDocPhibes committed:
Update Open.qex

AbomDocPhibes committed:
Update OpenPrompt.qex

AbomDocPhibes committed:
class replacement

AbomDocPhibes committed:
Update Open.adc

AbomDocPhibes committed:
prompt resource merge

AbomDocPhibes committed:
Update Open.adc

AbomDocPhibes committed:
Update config.xml

AbomDocPhibes committed:
Update Open.adc

AbomDocPhibes committed:
logic inclusion for prompt script

included a "None" option for useAI that will hide the prompt.css element entirely if selected. Changed logic for hiding Next button to only ever execute if useAI option is not "None"

AbomDocPhibes committed:
loop.html change, new adc build

AbomDocPhibes committed:
PropQuestion v7 fix

changed references to

CurrentADC.PropQuestion("exclusiveResponsesQuestion")

with

CurrentADC.PropValue("exclusiveResponsesQuestion")

AbomDocPhibes committed:
Update open_loop.html

AbomDocPhibes committed:
Update Open.adc

AbomDocPhibes committed:
propValue fix attempt

AbomDocPhibes committed:
propquestion fix attempt

AbomDocPhibes committed:
changed location of prompt options

AbomDocPhibes committed:
changed exclusive code from string back to question

AbomDocPhibes committed:
Update Open.adc

AbomDocPhibes committed:
exclusive response code troubleshooting

included a different version number for ease of testing between multiple developers

AbomDocPhibes committed:
Update Open.adc

gabrielpeter committed:
Update changelog.md

gabrielpeter commented on config.xml:
Can you rename aiprompt.css to prompt.css

gabrielpeter commented on resources/dynamic/default.html:
The class should be reverted to 'adc-open', styling depends on it

gabrielpeter commented on resources/dynamic/open-no-js.html:
The class should be reverted to 'adc-open', styling depends on it

gabrielpeter commented on resources/static/prompt.css:
Rename to prompt.css

gabrielpeter commented on resources/static/prompt.css:
Target your css more specific, add .adc-open to all of your styling, you never know there is some other p.bubble or .jumping-dots on the page

AbomDocPhibes committed:
naming and prompt.css changes

changed aiprompt.css to prompt.css in config and CSS files. added .adc-open tag to all styles in prompt.css

AbomDocPhibes committed:
class changes

changed classes from adc-prompt to adc-open, added adc-open div targeting bubble thought CSS class

AbomDocPhibes committed:
Update open_loop.html

AbomDocPhibes committed:
Update Open.adc

AbomDocPhibes committed:
Update Open.qex

AbomDocPhibes committed:
changed example qex files

removed prompts from open.qex and consolidated openprompt to be the main showcase of prompt capability to reduce confusion

gabrielpeter reviewed (approved):
LGTM
